ignite/handlers/time_limit.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import time
+from typing import Optional
+
+from ignite.engine import Engine
+
+log = logging.getLogger(__name__)
+
+__all__ = ["TimeLimit"]
+
+
+class TimeLimit:
+    """TimeLimit handler can be used to control training time for computing environments where session time is limited.
```

Comment:
Please, add another sentence here that "Timer starts when handler is created and not training started. This handler gracefully terminates the training if time passed in the training exceeds a limit."